src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TaskListViewModel.cs

```diff
@@ -28,6 +27,11 @@ public class TaskListViewModel : ApplicationSummaryViewModel
 
         public IEnumerable<ApplicationSequence> ApplicationSequences { get; set; }
         
+        public TaskListViewModel(IQnaApiClient qnaApiClient)
+        {
+            _qnaApiClient = qnaApiClient;
+        }
+
         public string CssClass(int sequenceId, int sectionId)
```

Comment:
CssClass by normal convention implies Cascading Style Sheets... can we put a comment on this to say it's not that?